File: Python Programming/problems on numbericals/problem27.py
'''
27.Write a program which accept accept range from user and display all numbers in between that range in reverse order.

	Author : Milind Nalawade
	
	Date : 27/06/2021
	
	Language : Python 3.9.0
'''

#====
#Logic
#====

def Range(min,max):
	if(max<min):
		print("Invalid range !!!\n");
	
	while(max>=min):
		print(max,end=" ");
		max-=1;
	


#=============
#Entry point function
#=============

def main():
	min=int(input("Enter your minimum value : "))
	max=int(input("Enter your maximum value : "))
	Range(min,max)
	
#====	
#stater 
#====
if __name__=="__main__":
	main()

File: C++ Programming/problems on numberical/problem27.cpp
/*

27.Write a program which accept accept range from user and display all numbers in between that range in reverse order.

	Author : Milind Nalawade
	
	Date : 27/06/2021
	
	Language : C++ programming
	
*/

using namespace std;
#include<iostream>

//Logic

void Range(int min,int max)
{
	if(max<min)
	{
		cout<<"Invalid range !!!\n";
	}
	
	while(max>=min)
	{
		cout<<max<<" ";
		max--;
	}
}

//Entry point function

int main()
{
	int i=0,j=0;
	
	cout<<" Enter your range : ";
	cin>>i>>j;
	Range(i,j);
}